wordpress-mobile/gutenberg-mobile
# Fix: Fix Gallery caption overlapped by other block

Fixes #2031 

Gallery caption gets covered by another block border.

Please also refer to:
[Related gutenberg PR](https://github.com/WordPress/gutenberg/pull/20987)
[Related gutenberg-mobile issue](https://github.com/wordpress-mobile/gutenberg-mobile/issues/2031)

To test:
1. Add `Gallery` block
2. Add gallery caption
3. Start adding new block, changing order and selection
4. See that Gallery caption is not being overlapped by next block border

PR submission checklist:
